packages/codd/tests/test_codd_hooks.py

```
@@ -538,8 +650,68 @@ def test_off_mode_bare_yaml_off_is_noop_even_with_errors(self, tmp_path: Path) -
+        _write(tmp_path, "docs/d.md", _CLEAN_DOC)
+        payload = {
+            "cwd": str(tmp_path),
+            "tool_name": "Bash",
+            "tool_input": {"command": 'git commit -m "msg"'},
+        }
+        result = _run_hook("codd-validate-precommit.py", payload, tmp_path)
+        assert result.returncode == 2  # index の壊れた内容で block される
+        assert "ブロック" in result.stderr
+
+    def test_warn_mode_ignores_unstaged_working_tree_errors(self, tmp_path: Path) -> None:
+        """index がクリーンなら、working tree だけの未ステージ変更（エラー含む）は無視する。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)